resources/assets/sass/components/_button.scss

```diff
@@ -16,20 +18,54 @@
 
     &:hover {
 
-        background: $color-fill-success-dark;
+        background-color: $color-fill-success-dark;
     }
 
     &.m-secondary {
 
+        @include color-modifiers-dark-and(color);
+
         color: $color-text-link;
         background: $color-white;
-        box-shadow: 0 1px 1px rgba($color-fill-neutral, 0.75);
+
+        &:hover {}
     }
 
     &.m-tertiary {
 
         color: $color-text-neutral-darker;
         background: $color-fill-neutral-light;
+
+        &:hover {
+
+            background: $color-fill-neutral;
+        }
+    }
+
+    &.m-border {
+
+        color: $color-white;
+        background: none;
+        box-shadow: inset 0 0 0 2px $color-white;
+
+        &:hover {
+
+            background: none;
+        }
+    }
+
+    &.m-icon {
+
+        padding: 0;
+        background: $color-white;
+
+        svg {
+
+            width: $spacer * 1.5;
+            height: $spacer * 1.5;
+
+            @include color-modifiers-and(fill);
```

Comment:
Expected item on line 67 to appear before line 64. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets